terraform/aws/modules/customer-web-server/web-server.tf

```diff
@@ -0,0 +1,391 @@
+provider "aws" {
+  region = var.region
+}
+
+data "terraform_remote_state" "cluster" {
+  backend = "s3"
+  config = {
+    bucket = "terraform-cloud-monitoring-state-bucket-${var.environment}"
+    key    = "mattermost-central-command-control"
+    region = "us-east-1"
+  }
+}
+
+resource "aws_security_group" "cws_postgres_sg" {
+  name                   = "cws_postgres_sg"
+  description            = "postgres SG for customer web server"
+  vpc_id                 = var.vpc_id
+  revoke_rules_on_delete = true
+
+  ingress {
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    security_groups = [data.terraform_remote_state.cluster.outputs.workers_security_group]
+  }
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = [var.cloud_vpn_cidr]
+    description = "CLOUD VPN"
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  tags = {
+    Name    = "postgres SG customer web server"
+    Created = formatdate("DD MMM YYYY hh:mm ZZZ", timestamp())
```

Comment:
maybe we can remove this to avoid update every time we run terraform

Reply:
@cpanato there is an ignore lifecycle below
`lifecycle {
    ignore_changes = [
      tags,
    ]`

Reply:
sweet
you are the master of terraform